sc/shared/src/test/scala/sigmastate/TestingInterpreterSpecification.scala

```diff
@@ -202,6 +203,21 @@ class TestingInterpreterSpecification extends CompilerTestingCommons
         |}""".stripMargin)
   }
 
+  property("Evaluate boolean casting ops") {
```

Comment:
This test is failing for v3 because `SBoolean.getMethods` (https://github.com/ScorexFoundation/sigmastate-interpreter/pull/932/files#diff-a26da11ac5b566197ea7da18de380e7f4b893212200c7a96d46be8869837c5a2R916) only evaluates once for all version test cases meaning it still has an empty `Seq()` for methods in v3 tests

I can't see why this is the case, maybe a global/thread var not getting reset between tests somewhere?

This `property` method is running the code block for all activated script/ergo tree version combinations under the hood